include/dungeon.h

```diff
@@ -27,7 +27,7 @@ typedef struct s_level { /* special dungeon level element */
     char boneid;    /* character to id level in bones files */
     uchar rndlevs;  /* no. of randomly available similar levels */
     d_flags flags;  /* type flags */
-    uchar which_level; /* TNNT: set to track which variant this is */
+    uchar variantno; /* TNNT: set to track which variant this is */
```

Comment:
I don't see a good reason to change this.

Reply:
I'll revert it -- I agree it certainly isn't necessary -- but I do still think `which_level` doesn't make it very clear what it refers to.  If I were in Minetown and someone asked me "which level are you on?" I would think they were expecting an answer like "Minetown" or "level 8", not "Minetown variant #2".  The comment in `struct s_level` is clear and precise though, and the way it's used makes it fairly clear too, so it's not as though anyone is likely to be genuinely led astray no matter what the name is.